rsmtool/configuration_parser.py

```diff
@@ -973,7 +977,7 @@ def parse(self, context='rsmtool'):
             A Configuration object containing the parameters in the
             file that we instantiated the parser for.
         """
-        _, extension = splitext(self._filename)
+        extension = Path(self._filename).suffix.lower()
         filepath = join(self._configdir, self._filename)
```

Comment:
```suggestion
        filepath = Path(self._configdir) / self._filename
```

Reply:
We should really just store `_configdir` as a `Path` object.

Reply:
I like it! Let me do that.

Reply:
Actually looks like `_configdir` is already a Path since it's currently defined as:

```
self._configdir = pathlike.resolve().parent
```